2022q4/status.adoc

```diff
@@ -13,10 +13,10 @@ In particular, the team is attempting to merge its GitHub repository into the Fr
 
 Here are the reasons for such a change:
 
-* having two independent repositories requires spending some time to make sure that both are in sync, which is being done manually. See for example commits such as
-link:https://github.com/freebsd/freebsd-quarterly/commit/4b8255e604dd0513e841aa8f3dce7741e78b999c[https://github.com/freebsd/freebsd-quarterly/commit/4b8255e604dd0513e841aa8f3dce7741e78b999c], which are not immediately clear in their commit messages about what is being done unless more time is invested to copy commit messages properly;
+* having two independent repositories requires spending some time to make sure that both are in sync, which is being done manually. See for example commits such as link:https://github.com/freebsd/freebsd-quarterly/commit/4b8255e604dd0513e841aa8f3dce7741e78b999c[https://github.com/freebsd/freebsd-quarterly/commit/4b8255e604dd0513e841aa8f3dce7741e78b999c], which are not immediately clear in their commit messages about what is being done unless more time is invested to copy commit messages properly;
 
-* the FreeSD doc/ repository is self-hosted, while the status repository is hosted on GitHub. Since the contents of the self-hosted repository are mirrored, nothing is being lost in visibility with the repository merging.
+* the FreeSD doc/ repository is self-hosted, while the status repository is hosted on GitHub.
+Since the contents of the self-hosted repository are mirrored, nothing is being lost in visibility with the repository merging.
```

Comment:
Maybe add a bit of space before "Since", so that it is clearer that this sentence is still part of the same list element?

Reply:
I can't remember which file, but before this PR I did notice superfluous white spaces at the beginnings of some lines beneath a bullet point. 

This creates ambiguity. When I first saw the artificial indent, my assumption was that the author intended to have a **separate paragraph**, indented, below the bullet point.

Reply:
You are right, indeed I created myself such a file: 2022q4/gcc.adoc. As you guessed, my intention was to create separate paragraphs there. And it did not work: white spaces at the beginnings of lines are just ignored, as you can see in the rendered version of the same file.

Then I suggest we do indent lines here. But we also need to find a solution to properly render those separated paragraphs: if you know the fix, please submit a PR for that too. Thank you very much.

Reply:
https://github.com/freebsd/freebsd-quarterly/pull/563 for 2022q4/gcc.adoc for your consideration

* if the approach to paragraphs looks good enough there, we might do something similar here. 

Thanks